src/stripe.android.ts

```diff
@@ -50,6 +51,53 @@ export class Stripe {
     );
   }
 
+  createSource(card: CardCommon, cb: (error: Error, source: Source) => void): void {
+    if (!card) {
+      if (typeof cb === 'function') {
+        cb(new Error('Invalid card'), null);
+      }
+      return;
+    }
+    const cardSourceParams = com.stripe.android.model.SourceParams.createCardParams(card.native)
+
+    try {
+      this._stripe.createSource(
+        cardSourceParams, 
+        new com.stripe.android.SourceCallback({
+          onSuccess: function(source) {
+              if (typeof cb === 'function') {
+                const newSource: Source = {
+                  id: source.getId(),
+                  amount: source.getAmount(),
+                  card: card,
+                  client_secret: source.getClientSecret(),
+                  created: source.getCreated(),
```

Comment:
After you change `created` to be a Date, the mapping will be `new Date(source.getCreated().toString())` (see how it's done in Token above).